integration_tests/features/Propagation.feature

```diff
@@ -60,34 +60,34 @@ Feature: Block Propagation
   @non-sync-propagation @long-running
   Scenario: Nodes should never switch to block sync but keep insync via propagation
     Given I have 1 seed nodes
-    Given I have a base node MINER connected to all seed nodes
+    Given I have full mining package MINER connected to all seed nodes
     And I have a lagging delayed node LAG1 connected to node MINER with blocks_behind_before_considered_lagging 10000
     Given I have a lagging delayed node LAG2 connected to node MINER with blocks_behind_before_considered_lagging 10000
         # Wait for node to so start and get into listing mode
     When I wait 100 seconds
-    When I mine 5 blocks on MINER
+    Given Mining node MINER mines 5 blocks
```

Comment:
This should be a `When`. `Given` is in the past tense and describes that something has been set up already. Either it should be `Given I have mined 5 blocks` or `When mining node MINER mines 5 blocks`